### DISCUSSION TITLE
Combobox bind FocusOut

### DISCUSSION THREAD

**Discussion by Megsman** [israel-dryer/ttkbootstrap#529] [General]
Is there an equivalent bind event for FocusOut for the Combobox?

`mycombobox = ttk.Combobox(...)`
`mycombobox.focus()`
`mycombobox.bind('<FocusOut>', do_something)`

... is not working.

Thanks.

**Comment by TheDemonGuard** [comment]
@Megsman 

With the attached script, I was not able to reproduce the issue (FocusOut will trigger the callback function).
[discussions_general_529.py](https://github.com/user-attachments/files/22871715/discussions_general_529.py)

Is it possible to retry with the latest ttkbootstrap version and check if the issue still persists?

In case the issue still persists,
Please provide working minimum reproducible demo/sample script.
And any necessary steps/instructions to reproduce the issue.
This will be helpful for anyone who will be interested in investigating the issue.

> Is there an equivalent bind event for FocusOut for the Combobox?
> 
> `mycombobox = ttk.Combobox(...)`
> `mycombobox.focus()`
> `mycombobox.bind('<FocusOut>', do_something)`
> 
> ... is not working.
> 
> Thanks.
